cmake/thirdparty/SetupGit.cmake

```diff
@@ -0,0 +1,47 @@
+###############################################################################
```

Comment:
Please put this directly in SetupThirdParty.cmake.  No reason to add an extra file for 5 lines.